fundation_app/views.py

```diff
@@ -20,8 +21,7 @@ def get_supported_organizations():
 
 
 def get_institutions():
-    # institutions = Institution.objects.order_by().values('name', 'description', 'type', 'categories__name', )
-    # .distinct()
+    # institutions = Institution.objects.all.values('name', 'description', 'type', 'categories__name', )
```

Comment:
missing parentheses -> Institution.objects.objects.all**()**.values()

please fix it and get all institutions with that command to avoid multiple database connections - you can filter it type later